### ISSUE TITLE
Usage of F1-socres

### ISSUE THREAD

**Issue by pferrariuy** [Renovus-Tech/solarec-python#19]
- [ ] Continuously review and analyze the model's performance using F1-score as the benchmark to achieve the predefined goal of accurate named entity extraction
- [ ] Implement further optimizations, fine-tuning hyperparameters, or model enhancements to improve the model's accuracy and bring the F1-score closer to the specific metric goal

**Comment by fcggamou** [comment]
This is no longer relevant since pivoting to LLM use for the NLP task. See https://github.com/Renovus-Tech/solarec-python/issues/14
